compiler/rustc_hir/src/lib.rs

```diff
@@ -39,6 +39,7 @@ pub mod weak_lang_items;
 #[cfg(test)]
 mod tests;
 
+pub use def::MacroKinds;
```

Comment:
Any specific reasons for this?
In other files I mostly see `MacroKinds` imported from `rustc_hir::def`, and it would be better to have just one "canonical location" for the type.

Reply:
@petrochenkov It's not required, it just seemed like a reasonable re-export given some of the other re-exports in `rustc_hir`, such as `pub use target::{MethodKind, Target};`.

I don't care strongly either way, though.